nextjs-13/src/lib/request/promiseWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+const promiseWrapper = (promise) => {
+  let status = 'pending';
+  let result;
+
+  const s = promise.then(
```

Comment:
Also spell out the complete variable name. Avoid cryptic variable naming: https://nimblehq.co/compass/development/code-conventions/#convention-12

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/110/commits/d30f9f526aeaccaa44ac7bfcc866291c1952f753#diff-f289ac85315a1af5de59576a60350c945a27574bc29040aa1f5607a01b0877fdR5)